Source/missiles.cpp

```diff
@@ -1355,7 +1355,14 @@ void AddMagmaball(int mi, int sx, int sy, int dx, int dy, int midir, char mienem
 	missile[mi]._mitxoff += 3 * missile[mi]._mixvel;
 	missile[mi]._mityoff += 3 * missile[mi]._miyvel;
 	GetMissilePos(mi);
+#ifdef HELLFIRE
+	if (missile[mi]._mixvel & 0xFFFF0000 || missile[mi]._miyvel & 0xFFFF0000)
+#endif
 	missile[mi]._mirange = 256;

```

Comment:
This gets a bit messy with the missing indentation, how about having an #else instead.

```
#ifdef HELLFIRE
 	if (missile[mi]._mixvel & 0xFFFF0000 || missile[mi]._miyvel & 0xFFFF0000) 
 	 	missile[mi]._mirange = 256; 
 	else 
 		missile[mi]._mirange = 1; 
 #else 
 	missile[mi]._mirange = 256; 
 #endif 
```

Reply:
yeah wasn't sure if I should go with else or two ifdefs, two ifdefs avoid code duplicating :P

Reply:
Yeah it's a value judgment :)

Reply:
applied